### DISCUSSION TITLE
SSH to turn off http authentication

### DISCUSSION THREAD

**Discussion by mcgibbaa** [rand256/valetudo#492] [Q&A] [answered]
I'm on the latest version and somehow lost / mistyped (?) the password when entering the info for the http authentication. I still have SSH access, but am not sure where to turn off and/or find the password via SSH.
Any suggestions?

**Comment by pidator** [comment, marked as answer]
Check the file `/mnt/data/valetudo/config.json`

**Reply by mcgibbaa** [reply]
That did it! Thanks.
